libafl_frida/src/alloc.rs

```diff
@@ -568,12 +569,12 @@ impl Allocator {
             &mut |range: &RangeDetails| -> bool {
                 let start = range.memory_range().base_address().0 as usize;
                 let end = start + range.memory_range().size();
-                if self.is_managed(start as *mut c_void) {
-                    log::trace!("Not unpoisoning: {:#x}-{:#x}, is_managed", start, end);
-                } else {
-                    log::trace!("Unpoisoning: {:#x}-{:#x}", start, end);
-                    self.map_shadow_for_region(start, end, true);
+                if start >= self.shadow_offset {
+                    log::trace!("Reached the shadow/allocator region - stopping");
+                    return false;
```

Comment:
I'm not 100% certain that there can be nothing allocated above the shadow/allocator region, under all circumstances. Can we not simply skip that region and continue afterwards if there is anything there? Seems safer?